lab_1_classify_by_unigrams/main.py

```diff
@@ -149,6 +270,22 @@ def collect_profiles(paths_to_profiles: list) -> list[dict[str, str | dict[str,
 
     In case of corrupt input arguments, None is returned
     """
+    if (not isinstance(paths_to_profiles, list)
+            or not all(isinstance(letter, str) for letter in paths_to_profiles)):
+        return None
+    list_for_dictionaries = []
+    for name_of_profile in paths_to_profiles:
+        if load_profile(name_of_profile) is None:
```

Comment:
you can replace it with `if not load_profile(name_of_profile):`

Reply:
done